tools/data_manipulation/xarray/xarray_mapplot.xml

```diff
@@ -0,0 +1,331 @@
+<tool id="xarray_mapplot" name="NetCDF xarray map plotting" profile="20.05" version="0.18.2">
+    <description>Visualize netCDF variables on a geographical map</description>
```

Comment:
You could include something like that:

```
    <macros>
        <token name="@TOOL_VERSION@">0.18.2</token>
        <token name="@VERSION_SUFFIX@">0</token>
    </macros>
```

This would also allow our new bot to bump the tool version automatically when there is a new xarray release.